multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -45,6 +63,14 @@ def best_quality():
     }
 
 
+@automm_presets.register()
+def high_quality_image_classification():
```

Comment:
Should this be `higher_quality`? Or change `high_quality_fast_inference_image_classification` to be good quality?

Reply:
This is an extra preset because previous benchmarking is done on swin-base. Ideally, we just want to have 3 per problem type.